public/app/plugins/datasource/tempo/traceql/TempoQueryBuilderOptions.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { AutoSizeInput, EditorField, EditorRow } from '@grafana/ui';
+import { QueryOptionGroup } from 'app/plugins/datasource/prometheus/querybuilder/shared/QueryOptionGroup';
+
+import { TempoQuery } from '../datasource';
+
+interface Props {
+  onChange: (value: TempoQuery) => void;
+  query: Partial<TempoQuery> & TempoQuery;
+}
+
+export const TempoQueryBuilderOptions = React.memo<Props>(({ onChange, query }) => {
+  const onLimitChange = (e: React.FormEvent<HTMLInputElement>) => {
+    onChange({ ...query, limit: parseInt(e.currentTarget.value, 10) });
+  };
+
+  return (
+    <>
+      <EditorRow>
+        <QueryOptionGroup title="Options" collapsedInfo={[`Limit: ${query.limit || 10}`]}>
```

Comment:
btw is this default in the data source code or something that is default in Tempo? If it's in data source code maybe we could change it to a constant and import it so it does not get out of sync accidentally.

Reply:
Good point. Just did some searching and realised that a default limit of 20 has been defined in a variable called `DEFAULT_LIMIT` [here](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/tempo/datasource.ts#L55).

Therefore, I should change the placeholder value here as the API will return 20 traces by default, not 10.

I will import `DEFAULT_LIMIT` and use that instead.

Reply:
https://github.com/grafana/grafana/pull/54723